{{cookiecutter.project_slug}}/Pipfile

```diff
@@ -32,3 +31,4 @@ google-cloud-secret-manager = "==2.12.6"
 google-auth = "==2.12.0"
 google-cloud-storage = "==2.5.0"
 pipenv = "*"
+drf-spectacular = "*"
```

Comment:
Would it be better to lock this to some semver range? Minor range should do the trick.

Reply:
Done!